DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -271,7 +271,8 @@
    <version ClassVersion="11" checksum="1239840459"/>
   </class>
 
-  <class name="pat::PackedCandidate" ClassVersion="19">
+  <class name="pat::PackedCandidate" ClassVersion="21">
+    <version ClassVersion="21" checksum="663492232"/>
```

Comment:
sure, the backports to 74X are going through a holding area. So, it's easy to not notice them.